wolfcrypt/test/test.c

```diff
@@ -1981,6 +1982,18 @@ WOLFSSL_TEST_SUBROUTINE int md2_test(void)
             return -1500 - i;
     }
 
+    for (i = 0; i < times; ++i){
```

Comment:
Minor style request - please add a space after the closing ')' in your for loop here, and if statements below.  Thanks!

Reply:
For future reference, please allow the original reviewer to mark an issue as "Resolved", thanks!